src/Vehicle/Vehicle.h

```diff
@@ -1030,6 +1030,8 @@ private slots:
     void _chunkedStatusTextCompleted    (uint8_t compId);
     void _setMessageInterval            (int messageId, int rate);
     EventHandler& _eventHandler         (uint8_t compid);
+    // to load metat DLB Error code Description
```

Comment:
Please add more details what exactly is the metadata and what the dunction is for